src/main/java/org/primefaces/component/imagecropper/ImageCropperRenderer.java

```diff
@@ -180,6 +184,15 @@ public Object getConvertedValue(FacesContext context, UIComponent component, Obj
         Resource resource = getImageResource(context, cropper);
         InputStream inputStream = null;
         String imagePath = cropper.getImage();
+        Object streamObject = cropper.getStream();
+        StreamedContent stream = null;
+        if (streamObject != null && !(streamObject instanceof StreamedContent)) {
```

Comment:
I don't think you have check != null because intanceof NULL is always false. you can invert this by checking is streamobject instanceof ELSE throw exception

Reply:
The check `streamObject != null` is performed because '`cropper.getStream()` null' is a valid condition if '`cropper.getImage()` is not null'. I just included some tests for check these scenarios.

Reply:
I think @tandraschko wants you to not introduce a new `stream` property and just re-use `image` so it can either be a URL or StreamedContent like GraphicImage does it.